### ISSUE TITLE
Consider disabling `RCS0012` for fields

### ISSUE THREAD

**Issue by RenderMichael** [dotnet/roslynator#1395]
**Product and Version Used**: 

4.10.0

**Steps to Reproduce**:

``` csharp
internal sealed class C
{
    private readonly string? x; // RCS0012
    private readonly string? y;

    public C(string x, string y)
    {
        this.x = x;
        this.y = y;
    }

    public void UseVariables()
    {
        // dismiss warnings about unused variable
        _ = x;
        _ = y;
    }
}
```

**Actual Behavior**:
Warning `RCS0012`: Add blank lines between single-line declarations

**Expected Behavior**:
In contrast with properties and methods, fields are often written without a separate line between them. There is a case to be made that the analyzer should not warn in this case.

Perhaps it would at least be possible to make a different diagnostic for fields, so it can be separately suppressed.

**Comment by josefpihrt** [comment]
Maybe the desired behavior could be achieved by adding editorconfig option, something like:

```editorconfig
roslynator_blank_line_between_singleline_declarations.field = false
```

**Comment by salixzs** [comment]
I would add here a possibility to configurable-disable this for enum definitions.
```csharp
public enum MyEnum
{
    Default = 0,  // RCS0012
    SomeValue1 = 1,
    SomeValue2 = 2,
    SomeValue3 = 3
}
```
